frontend/src/HeroSection.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+export default function HeroSection({title}) {
```

Comment:
curious, what does "Hero" stand for?

Reply:
The entire section on top: 

Review:
<img width="1469" alt="Screenshot 2024-07-26 at 6 32 28 PM" src="https://github.com/user-attachments/assets/4e87e3ee-851a-4dbb-9a17-8653b537b3e9">